src/css/main.less

```diff
@@ -838,36 +838,25 @@ fieldset[disabled] .btn-info.active {
   background-color: #5e646b;
 }
 
-.health-dot {
-  display: inline-block;
-}
 
-.health-dot:before {
-  content: "\00a0";
-  border-radius: 50%;
-  border: solid 2px #51565d;
-  width: 12px;
-  height: 12px;
-  display: inline-block;
-  vertical-align: bottom;
-  margin-right: 6px;
-  margin-bottom: 1px;
-}
+/* ==============
```

Comment:
I agree with @orlandohohmeier, it doesn't hurt in light of a future refactoring and will speed up extraction.